### ISSUE TITLE
Make the footer in the main app prettier

### ISSUE THREAD

**Issue by micahdbak** [CSSS/csss-site-frontend#40]
First note, about the address:
- Darrick says it looks gross, make it look nice I guess

Gabe wants to:
- Add pixel art or something
- Put more info in it
- Generally make it more pretty!

See:
- Footer of w3schools: https://www.w3schools.com/sql/default.asp

**Comment by micahdbak** [comment]
@EarthenSky - I'm assigning this issue to you - make the footer pretty!!

The component is in /src/main/components/Footer.js

Go crazy!

**Comment by EarthenSky** [comment]
> Darrick says it looks gross

Darrick is right, I noticed that too. We should steal from existing websites. What I found is that `Contact` is commonly used, with one example of formatting [here ](https://www.safe.com/contact/#:~:text=Safe%20Headquarters-,Safe%20Software,-Suite%201200%20%E2%80%93%209639), and [two](https://www.relentlesstechnology.com/company.html?gad_source=1&gclid=EAIaIQobChMI4LHQ6YDwhwMVhQutBh2oljgzEAAYAiAAEgKdafD_BwE#:~:text=206%20%2D%20197%20Forester%20Street) [more examples](https://www.sfu.ca/contact/burnaby.html#:~:text=8888%20University%20Drive).

**Comment by micahdbak** [comment]
@EarthenSky progress?

**Comment by EarthenSky** [comment]
none yet

**Comment by micahdbak** [comment]
hurry up >:(

**Comment by micahdbak** [comment]
(JK take ur time, im just bugging everyone for updates :) )

**Comment by micahdbak** [comment]
@EarthenSky - still wanna do this?

**Comment by EarthenSky** [comment]
Yeah, but it's not a priority compared with getting the backend complete. If someone else is interested feel free to give it to them!

**Comment by micahdbak** [comment]
ahh gotcha, all good then!

**Comment by jbriones1** [comment]
I'll probably put the contact information in the navigation bar, unless there's a better reason to place it at the bottom of each page.
